src/middlewares/index.ts

```diff
@@ -0,0 +1,2 @@
+export { createInsightsMiddleware } from './insights';
+export * from './createRouter';
```

Comment:
We exports the types from the router middleware. Any opinions on aligning the two? (either exporting the types or not exporting the types)

Reply:
I think it's good to export all of them https://github.com/algolia/instantsearch.js/pull/4446/commits/e7c7e5858aada1f4ee105764d524ca0bc54212ed